### DISCUSSION TITLE
Gotify notification support

### DISCUSSION THREAD

**Discussion by Crupuk** [pymedusa/Medusa#8039] [Ideas & Feature requests]
Hi,

Thanks for your work ;-) 

Could you please considere to  implement [Gotify](https://gotify.net/) notification or at least the possibility to call a script for sending notification .

**Comment by plato79** [comment]
Just FYI,

The message format is like this if called via curl:

curl "https://<message_server>/message?token=<app_token>" -F "title=my title" -F "message=my message
" -F "priority=5"

So, the only requirement is message server URL and "application token" created in the server.

I'll look into it. But I'm not a pro at python and someone already worked on this could make this faster.

**Reply by plato79** [reply]
Just a request..

If nobody can look into it, could you at least give me pointers on which files should be modified and how?

I know I need to add a new `gotify.py` file ( in `/medusa/notifier` folder ), configured to send the request. I think creating the file based on `freemobile.py` would make it easier ( the sending process seems similar ). Of course there are additional modifications needed in `__init__.py` under `/medusa/notifier` folder.

Also need modifications under `/medusa/server/web/config/notifications.py`

Though I couldn't find any reference to notifier image. I know it somehow gets icon.notifier.<notifier> but from where and how?

Any help is appreciated.
